client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.component.js

```diff
@@ -0,0 +1,10 @@
+import controller from './pack-xdsl-access-comfort-exchange.controller';
+import template from './pack-xdsl-access-comfort-exchange.html';
+
+angular.module('managerApp').component('xdslAccessComfortExchange', {
```

Comment:
It's not a lot there?

Reply:
What do you mean ?

Reply:
I think it's a lost of time for finally adding "angular.module('managerApp').component(MyComponent)". Sorry but there it's too much.

Reply:
It may be a little lost of time, ok. But :

- Having every module cutted properly make it clearer to evolve, maintain. Even if you change location of the module.
- Plus, if we want to migrate from angularjs to whatever framework, it will be simpler to do it, as we reduce places where we declare component/module into angular app, instead of searching in every controller/component/service files for `angular.module('managerApp')` and remove it.
- Last point, it is what we tend to do for all of our codebase, I understand it might be frustrating to adopt new style of coding, but let's all make little effort to harmonize all our applications.